account_payment_partner/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     "name": "Account Payment Partner",
-    "version": "18.0.1.0.1",
+    "version": "18.0.2.0.0",
```

Comment:
Don't change this, and the migration script should be in `18.0.1.0.0` folder. If you want to trigger this on your migrated DB, execute in a shell or reset the latest version to a previous one.